examples/investment.jl

```diff
@@ -58,8 +58,9 @@ function create_nodes()::DD.Tree
     #subproblem formulation
     function subproblem_builder(tree::DD.Tree, subtree::DD.SubTree, node::DD.SubTreeNode)
         mdl = subtree.model
-        x = @variable(mdl, x[l=1:L], Int, base_name="n1_x")
-        #x = @variable(mdl, x[l=1:L], base_name="n1_x")
+        #x = @variable(mdl, x[l=1:L], Int, base_name="n1_x")
+        x = @variable(mdl, x[l=1:L], base_name="n1_x")
```

Comment:
Can you use anonymous variable here? Something like this?
```julia
x = @variable(mdl, [l=1:L], base_name="n1_x")
```

Reply:
resolved by JuMP.jl file